spec/services/ccms/case_add_requestor_xml_blocks_spec.rb

```diff
@@ -846,6 +849,189 @@ module CCMS # rubocop:disable Metrics/ModuleLength
             expect(block).to have_response_value 'false'
           end
         end
+
+        context 'attributes hard coded to false' do
+          it 'should be type of text hard coded to false' do
+            attributes = [
+              [:global_means, 'COST_LIMIT_CHANGED_FLAG']
+            ]
+            attributes.each do |entity_attribute_pair|
```

Comment:
Why are you using a block here when there is only one set of attributes being tested? Just do:
```ruby
block = XmlExtractor.call(xml, :global_means, 'COST_LIMIT_CHANGED_FLAG')
```

Reply:
this should actually be a block of 2 tests, global_merits and gloabl_means, fixed